syntax = "proto3";

message TestProto {
  bool test_bool = 1;
  int32 test_int = 2;
}

